test/demo/utils/toastUtils.ts

```diff
@@ -0,0 +1,29 @@
+import {ElementHandle} from 'puppeteer';
+import {Identity, Window} from 'hadouken-js-adapter';
+
+import {SERVICE_IDENTITY} from '../../../src/client/internal';
+
+import {OFPuppeteerBrowser} from './ofPuppeteer';
+import {fin} from './fin';
+
+export function getToastIdentity(sourceApp: Identity, notificationId: string): Identity {
+    return {uuid: SERVICE_IDENTITY.uuid, name: `Notification-Toast:${sourceApp.uuid}:${notificationId}`};
+}
+
+export async function getToastWindow(sourceApp: Identity, notificationId: string): Promise<Window | undefined> {
+    const toastIdentity = getToastIdentity(sourceApp, notificationId);
+    const childWindows = await fin.Application.wrapSync(SERVICE_IDENTITY).getChildWindows();
+    return childWindows.find(win => win.identity.uuid === toastIdentity.uuid && win.identity.name === toastIdentity.name);
+}
+
+const ofBrowser = new OFPuppeteerBrowser();
+export async function getToastCards(sourceApp: Identity, notificationId: string): Promise<ElementHandle[] | undefined> {
```

Comment:
I think it's safer to not rely on "there should only be one" for integration tests. In fact, I explicitly decided to return an array so we could be absolutely sure there was only one card, rather than taking it on faith.

Reply:
Fair enough 👍